Update pages around usage (#4787)
Qiskit/documentation: docs/guides/execution-modes-faq.mdx

Batch mode:

> - The total run time is less because the classical processing might run in parallel.

Author: ```suggestion
- The total run time is less because the classical processing might run in parallel.
- There is a slight overhead for running each job, so you end up paying a little more for batched jobs. This overhead correlates to the size of the job.
- Because batch mode doesn't give you exclusive access to a backend, jobs inside a batch might run with other users' jobs or calibration jobs.
- If some jobs fail, you still get results from the completed jobs.
- You can take action in the middle of a batch workload based on the results of completed jobs. For example, you can cancel the rest of the jobs if the initial results look incorrect.
```

Batch mode:

- The total run time is less because the classical processing might run in parallel.
- There is a slight overhead for running each job, so you end up paying a little more for batched jobs. This overhead correlates to the size of the job. For example, the total usage of two jobs, each containing 40 100x100 circuits, is six seconds more than a single job containing 80 circuits.
- Because batch mode doesn't give you exclusive access to a backend, jobs inside a batch might run with other users' jobs or calibration jobs.
- If some jobs fail, you still get results from the completed jobs.

> - You can take action in the middle of a batch workload based on the results of completed jobs. For example, you can cancel the rest of the jobs if the initial results look incorrect.

Author: It's been so long that I don't recall 🙃 @francabrera were you the one who asked we remove these bullets? If they are still true, I will restore them

> - You can take action in the middle of a batch workload based on the results of completed jobs. For example, you can cancel the rest of the jobs if the initial results look incorrect.

Author: Restoring, in discussion with the team

> - There is a slight overhead for running each job, so you end up paying a little more for batched jobs.

Reviewer: ```suggestion
- There is a slight overhead for running each job, so you end up cosuming a little more for batched jobs.
```

Batch mode:

- The total run time is less because the classical processing might run in parallel.

> - There is a slight overhead for running each job, so you end up paying a little more for batched jobs. This overhead correlates to the size of the job.

Reviewer 2: ```suggestion
- There is a slight overhead for running each job, so you end up paying a little more for batched jobs versus when all circuits are executed in a single job.
```
